src/App.tsx

```diff
@@ -1,25 +1,12 @@
 import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+import ToDo from './components/ToDo';
```

Comment:
use absolute path here and define "src" folder as rootDirectly. So, it will convert to
import ToDo from 'components/ToDo';